_config/config.yml

```diff
@@ -0,0 +1,9 @@
+---
+Name: silverstripe-colourpicker
+---
+
+SilverStripe\Core\Injector\Injector:
+  Colour:
+    class: MadeHQ\FieldType\DBColour
+  Color:
```

Comment:
Get rid. We're not Americans, SilverStripe isn't American.

Reply:
Done